package.json

```diff
@@ -249,6 +258,11 @@
                     "command": "appService.Refresh",
                     "when": "view == azureAppService",
                     "group": "navigation@2"
+                },
+                {
+                    "command": "appService.Deploy",
+                    "when": "view == azureAppService",
+                    "group": "navigation@3"
```

Comment:
I would put this between create and refresh. Refresh is consistently the last command for all our extensions